test/functional/hooks.js

```diff
@@ -42,37 +43,126 @@ describe('library hooks', () => {
     }
 
     beforeEach(() => {
-      sandbox.spy(attributes, 'applyAttr');
+      stub = sinon.stub();
```

Comment:
I was using a stub since there wasn't a function there to spy on. But removing the ability to null out the generic mutator makes everything functions now.